tests/slate/test_hdg_poisson.py

```diff
@@ -50,7 +51,10 @@ def run_LDG_H_problem(r, degree, quads=False):
     f = Function(Vh).interpolate(-div(grad(a_scalar)))
 
     # Stability parameter
-    tau = Constant(1)
+    # FIXME: This could just be a Constant, but something breaks in SCPC when
+    # the test suite is run out of order. Specifically different code is
+    # generated on different ranks since constants are not renumbered correctly.
+    tau = Function(R).assign(1)
```

Comment:
Make an issue and assign me?

Reply:
#3802 Is it alright to change this back now for the purposes of the PR?

Reply:
Sure

Reply:
Your other PR fixed this